scripts/stage-release.sh

```diff
@@ -60,6 +61,16 @@ EOF
   esac
 done
 
+if [[ -z "${GPG_KEY}" ]]; then
+    echo "GPG_KEY not set unable to sign the release. Please specify -k <YOUR_GPG_KEY>" 1>&2
+    exit 1
+fi
+
+if [[ -z ${RELEASE_VERSION} ]]; then
+  echo "No version specified aborting"
```

Comment:
Nit: add a comma to make the message clearer
```suggestion
  echo "No version specified, aborting"
```